### PR TITLE
Consistent styling and refactoring components

### PR DESCRIPTION
**Pull request by SowmyaVijayendra**

### CODE REVIEW

File: react-front-end/src/components/MovieDetails.js [hunk 1]
```diff
@@ -15,15 +15,15 @@ function MovieDetails(props) {
...
 
   useEffect(() => {
-    if(typeof currentMovieDetails !== "undefined"){
+    if (typeof currentMovieDetails !== "undefined") {
       const { fireFunction } = location.state;
-      if(fireFunction){
+      if (fireFunction) {
         handleWatchTrailerClick(currentMovieDetails.original_title);
       }
-      window.scrollTo(0,10000)
```
Comment (sl-nair) [inline react-front-end/src/components/MovieDetails.js:26]: This was to scroll down to the trailer when you click "watch-trailer" from the banner.

**Review by sl-nair — CHANGES_REQUESTED** [review]

File: react-front-end/src/components/MovieDetails.js (hunk 1 above)
Comment (SowmyaVijayendra) [reply react-front-end/src/components/MovieDetails.js:26]: The above code gets executed everytime user navigates to movie details page ireespective of whether user clicked on watch trailer or just to view info. To achieve your requirement, i have made code fixes to pushed the changes now.

**Review by myfootsasleep — APPROVED** [review]
looks good to me

File: react-front-end/src/components/MovieDetails.js (hunk 1 above)
Comment (myfootsasleep) [reply react-front-end/src/components/MovieDetails.js:26]: will resolve this for now, so i can merge with master

**Pull request merged**